src/main/java/ch/njol/skript/sections/EffSecSpawn.java

```diff
@@ -55,7 +53,9 @@
 	"Do note that other event values, such as 'player', won't work in this section.",
 	"",
 	"If you're spawning a display and want it to be empty on initialization, like not having a block display be stone, " + 
-	"set hidden config node 'spawn empty displays' to true."
+	"set hidden config node 'spawn empty displays' to true.",
+	"",
+	"Note that when spawning an entity via entity snapshots, the code within the section will not run instantaneously as compared to normal."
```

Comment:
```suggestion
	"Note that when spawning an entity via entity snapshots, the code within the section will not run instantly, as compared to normal entity spawning."
```
what does "not run instantly" mean here, exactly? should clarify if possible

Reply:
Sorry, but changing `instantaneously` to `instantly` does not sound right. 
Not sure how to "clarify" without explaining how the process works between spawning via snapshots vs entitydata.
"Instantaneously" clarifies as best and simple saying that the code will run a bit later when spawning a snapshot

Reply:
<img width="185" alt="image" src="https://github.com/user-attachments/assets/0597d5d0-e62b-490a-a74b-fec02c7fe088">

Reply:
Right,  I know they are basically the same, but "instantaneously", to me atleast, seems to fit more than "instantly". Smoother sentence